cluster/gce/trusty/master.yaml

```diff
@@ -213,6 +213,7 @@ script
 	set -o errexit
 	set -o nounset
 
+	. /etc/kube-configure-helper.sh
```

Comment:
Correct. I guess this has diverged significantly from master, since I don't see a block on the master branch with `filename="kube-addons.conf"`.